images/openshift-golang-builder.Dockerfile

```diff
@@ -48,6 +48,11 @@ RUN yum update -y && \
     mkdir -p /go/src && \
     yum clean all -y
 
+# patch in a build tag to enable sloppy ipv4 parsing per https://issues.redhat.com/browse/ART-3664
+COPY go-1.16-lib.patch .
+RUN patch -ruN --strip=1 -d /usr/lib/golang/ < go-1.16-lib.patch && rm -f go-1.16-lib.patch
+
+# provide a cross-compiler for windows/mac binaries (amd64 only)
```

Comment:
fwiw since this isn't the newest version, this comment won't be carried over to future versions either.